nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1129,8 +1127,17 @@ define([
          * Render the UI according to the FSM
          */
         function renderUI() {
+            if (!ui) {
```

Comment:
this should never happen IRL, but since it happened to me whilst testing, I added a bit of checking and error reporting.